api/v1alpha1/osbuildconfig_types.go

```diff
@@ -63,6 +63,17 @@ type BuildDetails struct {
 	TargetImage TargetImage `json:"targetImage"`
 }
 
+// EdgeInstallerBuildDetails includes all the information needed to build the edge-installer image
+type EdgeInstallerBuildDetails struct {
```

Comment:
Since this type is only used in the OSBuildSpec, its definition should move to the osbuild_types.go file